packages/react/src/components/searchInput/SearchInput.tsx

```diff
@@ -94,8 +94,10 @@ export const SearchInput = <SuggestionItem,>({
   const inputRef = useRef<HTMLInputElement>(null);
   const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
   const [inputValue, setInputValue] = useState<string>('');
+  const [isItemClicked, setIsItemClicked] = useState<boolean>(false);
```

Comment:
This could probably be a ref too? 🤔

Reply:
Ref might be better if there's no need to re-render the component?

Reply:
- [ ] use ref instead of useState